source/base/image/openexr.cpp

```diff
@@ -46,21 +46,11 @@
 
 #ifndef OPENEXR_MISSING
 
-#ifdef NON_UNIX_OPENEXR_HEADERS
-
 #include <ImfRgbaFile.h>
 #include <ImfStringAttribute.h>
 #include <ImfMatrixAttribute.h>
 #include <ImfArray.h>
 
-#else
-
-#include <OpenEXR/ImfRgbaFile.h>
-#include <OpenEXR/ImfStringAttribute.h>
-#include <OpenEXR/ImfMatrixAttribute.h>
-#include <OpenEXR/ImfArray.h>
-
-#endif
```

Comment:
Given that removing this seems to work fine, I'm stumped by the question, _how did that not cause any issues before?_

If you have the time, could you perhaps dig into the `windows/vs10/povbase.vcxproj` XML file, and remove any reference to `NON_UNIX_OPENEXR_HEADERS`?